cnames_active.js

```diff
@@ -268,7 +268,7 @@ var cnames_active = {
   "bachors": "bachors.github.io",
   "backlog": "backlog-js.github.io/backlog.js.org", // noCF? (don´t add this in a new PR)
   "backstage": "backstagejs.github.io/backstage",
-  "badger": "just-glue-it.github.io/badger", // noCF? (don´t add this in a new PR)
+  "badger": "just-glue-it.github.io/badgerld", // noCF? (don´t add this in a new PR)
```

Comment:
```suggestion
  "badger": "just-glue-it.github.io/badger", // noCF? (don´t add this in a new PR)
```
Please revert the unrelated change here